files/conf/ipk.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Why don't you change "files/conf/ipk-meta.json?"

Reply:
We decided to keep it as it is.